### ISSUE TITLE
Add individual records to saved or archived searches?

### ISSUE THREAD

**Issue by AJLinn** [ArctosDB/arctos#2580]
Issue Documentation is http://handbook.arctosdb.org/how_to/How-to-Use-Issues-in-Arctos.html

**Is your feature request related to a problem? Please describe.**
Is it possible to add individual object records to saved/archived searches?

** Describe what you're trying to accomplish** 
I created this archived search (http://arctos.database.museum/archive/modelboats_alltypes) by doing a search for "model" and then removed all the items I didn't want included in the results, then saved that search.
Today I just fixed a catalog number because it was not totally accurate (added the "AJ" to this catalog number http://arctos.database.museum/guid/UAM:EH:0317-0002AJ).  Now that record does not show up in the archived search. Can I add it back in somehow? Would it have still been included if I had done this as a saved search instead of archived?

**Describe the solution you'd like**
I'd like to be able to add records to established saved or archived searches if I discover one that is relevant but not previously included.

**Describe alternatives you've considered**
As far as I can tell, I'd have to totally re-create this set of custom results

**Comment by dustymc** [comment]
>add it 

<img width="693" alt="Screen Shot 2020-04-03 at 3 28 38 PM" src="https://user-images.githubusercontent.com/5720791/78409914-d28ed200-75bf-11ea-87b4-017548f8cf31.png">

``+modelboats_alltypes`` should do it.

> saved search instead

Depends on the criteria used to create the saved search - saved searches are just whatever matches search terms when the page is opened, Archives are essentially lists of GUIDs.

http://handbook.arctosdb.org/documentation/archive.html

**Comment by AJLinn** [comment]
> Archives are essentially lists of GUIDs

So if the GUID changes (like by adding AJ to the end of my catalog number) it no longer matches the list of GUIDs, correct? I should re-do that search as a saved search?

**Comment by dustymc** [comment]
Yes, the expectation is that the GUID won't change. Adding a redirect might be useful if there's any possibility of external links. The + trick can easily add it back in.

The choice between an Archive and saved search depends on the goals - do you need a static set of records, or whatever matches some criteria at runtime?

**Comment by krgomez** [comment]
Is there a way to remove catalog records from an archived search? I cannot see how to do that.

**Comment by dustymc** [comment]
>remove

No - but you should be able to search/delete the archive/create a new archive of the old name.
